spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  let(:user) do
+    User.create(name: 'Harry', photo: 'https://photos.com/harrystyles', bio: 'harry Styles here!')
+  end
+
+  describe 'Get /' do
+    it 'Sends a success response' do
+      get '/'
+
+      expect(response).to have_http_status(:success)
+    end
```

Comment:
- [ ] Below tests are failing, Please make sure that all tests pass without any errors.

![image](https://github.com/simplegoose/blog_app/assets/83803414/500eae72-0ad5-4082-b807-6f907f984595)